web/src/components/HeaderNavigationItem.tsx

```diff
@@ -58,22 +77,31 @@ const StyledLink = styled(Link)<{ $active: boolean }>`
       : ''}
 `
 
+const StyledText = styled.span<{ $active: boolean }>`
+  @media ${dimensions.smallViewport} {
+    ${props => (props.$active ? 'font-weight: 800' : '')}
```

Comment:
Where is the text set to bold, if active, for bigger screens? It makes much more sense to set it in the same file for both screen sizes.

Reply:
fixed